packages/terra-clinical-header/src/Header.jsx

```diff
@@ -35,40 +35,138 @@ const defaultProps = {
   isSubheader: false,
 };
 
-const Header = ({ title, startContent, endContent, isSubheader, ...customProps }) => {
-  let startElement;
-  if (startContent) {
-    startElement = <div className={cx('flex-end')}>{startContent}</div>;
+class Header extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleOnResize = this.handleOnResize.bind(this);
+    this.state = {
+      startContentInitialWidth: undefined,
+      endContentInitialWidth: undefined,
```

Comment:
I don't think these initial widths need to be stored in the state. State updates triggers the component to rerender and we shouldn't need to do that in order to set the initialWidth. Just set `this.startContentInitialWidth` and `this.endContentInitialWidth` in componentDidMount instead.

Reply:
removed these two items